### ISSUE TITLE
RedNotebook window becomes undisplayable after you roll up the window in Xfce

### ISSUE THREAD

**Issue by jendrikseipp** [jendrikseipp/rednotebook#111]
Ubuntu 9.04, Xfce.

Open up RedNotebook, then roll up the window (press on the up-pointed arrow in title bar) and click on RedNotebook tray icon.
The rolled up window will not disappear, but if you try to roll the "minimzed-via-tray" down to its full size, it disappears and won't reappear again until you restart the program.

Screenshot of the "rolled-up" window included (just in case).

**Comment by jendrikseipp** [comment]
*Original comment by* **a-ztech** (2010-02-21T08:46:15Z):

----------------------------------------

**Comment by jendrikseipp** [comment]
*Original comment by* **jendrikseipp** (2010-02-22T02:57:24Z):

----------------------------------------

Unfortunately I don't have an XFCE installation at hand so I cannot reproduce this error. Unless there is somebody who can tell me what has to be changed for it to work on XFCE, I guess we will have to wait a bit and see when I find the time to install an XFCE environment in virtualbox (don't want to take chances on ruining my system here ;)

**Comment by jendrikseipp** [comment]
*Original comment by* **a-ztech** (2010-02-23T13:31:13Z):

----------------------------------------

Thanks for paying attention. =)
Some information about system: I'm using Ubuntu 9.04 on Pentium 4 with "xfce-desktop" package set up on it. Not Xubuntu distribution.

**Comment by jendrikseipp** [comment]
*Original comment by* **jendrikseipp** (2010-04-29T19:18:25Z):

----------------------------------------

Hi, could you try to send me a patch? I don't have much time to investigate this issue.

**Comment by jendrikseipp** [comment]
*Original comment by* **a-ztech** (2010-04-30T12:01:22Z):

----------------------------------------

Sorry, not a programmer. -.-

**Comment by jendrikseipp** [comment]
*Original comment by* **jendrikseipp** (2010-11-26T01:21:40Z):

----------------------------------------

I have changed the minimize / tray code a little bit. Does the error still happen with the latest version from trunk (bzr branch lp:rednotebook)?

**Comment by jendrikseipp** [comment]
*Original comment by* **a-ztech** (2010-11-26T20:36:42Z):

----------------------------------------

Hm, no. It is still not fixed.


On the first try, the behaviour was quite new. When I rolled the window up, clicked the tray icon and then rolled the window down, it displayed and empty unfilled program frame (no contents, just the usual GTK borders). When I rolled the window back up, clicked the tray icon the second time and rolled it down. it worked successfully. 

After restart, it started to show the same bug as in the first message.

Console shows nothing: it happily announces whatever it does usually.
